app/models/api/trainee_attributes/v01.rb

```diff
@@ -63,7 +62,11 @@ class V01
       attribute :record_source, default: -> { RecordSources::API }
 
       validates(*REQUIRED_ATTRIBUTES, presence: true)
+      validates :email, presence: true, length: { maximum: 255 }
 
+      validate do |record|
+        EmailFormatValidator.new(record).validate
+      end
```

Comment:
Could we have some spacing here :trollface:

Reply:
fixed https://github.com/DFE-Digital/register-trainee-teachers/pull/4150/commits/d09f3ee97ec99266fca76c701c59812cc6983367